monkey/monkey_island/cc/resources/reporting/report.py

```diff
@@ -31,11 +33,16 @@ def get(self, report_type=SECURITY_REPORT_TYPE, report_data=None):
                     "statusesToPillars": ZeroTrustService.get_statuses_to_pillars(),
                     "pillarsToStatuses": ZeroTrustService.get_pillars_to_statuses(),
                     "grades": ZeroTrustService.get_pillars_grades()
-                }
-                )
+                })
             elif report_data == REPORT_DATA_PRINCIPLES_STATUS:
                 return jsonify(ZeroTrustService.get_principles_status())
             elif report_data == REPORT_DATA_FINDINGS:
-                return jsonify(ZeroTrustService.get_all_findings())
+                return jsonify(FindingService.get_all_findings())
```

Comment:
A better abstraction would be calling `get_all_findings()` from `ZeroTrustService`. Why not have just 1 service export all the ZT report functionalities?

Reply:
What would that file be, just a mapping to relevant ZeroTrustService and ZTFindingService methods? I see no problem with resource specifying which data to return and calling different services as long as it's not processing the data.

Reply:
I thought about it and I agree.